docs/tutorial/part-four/index.md

```diff
@@ -354,26 +336,20 @@ export default ({ children }) => {
 // highlight-end
 ```
 
-Another success! 🎉
+एक और सफलता! 🎉
 
 ![Page title and layout title both pulling from siteMetadata](site-metadata-two-titles.png)
```

Comment:
`Page title and layout title both pulling from siteMetadata` => `पेज शीर्षक और लेआउट शीर्षक दोनों siteMetadata से पुल्ल कर रहे हैं`

Reply:
changed ```पुल्ल ``` to ```पुल```